archive/demos/subtractive.html

```diff
@@ -518,6 +506,12 @@
     // As one of our waveform types, use a nice organ.
     loadWaveTable("wave-tables/Twelve_OpTines");    
 }
+async function resumeCtx(){
+    if(context.state!='running') await context.resume();
+    
+    lfo.start(0);
+        navigator.requestMIDIAccess().then(onMIDISuccess, onMIDIFailure);
```

Comment:
Since this scope is in `async`, we might want to use `await` instead of using Promise.